src/modules/commander/commander_params.c

```diff
@@ -513,10 +513,9 @@ PARAM_DEFINE_INT32(COM_ARM_AUTH_MET, 0);
 PARAM_DEFINE_FLOAT(COM_ARM_AUTH_TO, 1);
 
 /**
- * Horizontal position error threshold.
+ * Horizontal position error threshold for multicopter.
  *
  * This is the horizontal position error (EPH) threshold that will trigger a failsafe.
```

Comment:
Let's link `EKF2_NOAID_TOUT` here - as usually that one hits before `COM_POS_FS_EPH`.

Reply:
addressed